examples/nvidia_dali/README.md

```diff
@@ -2,10 +2,51 @@
 
 The NVIDIA Data Loading Library (DALI) is a library for data loading and pre-processing to accelerate deep learning applications. It provides a collection of highly optimized building blocks for loading and processing image, video and audio data.
 
-In this example, we use NVIDIA DALI for pre-processing image input for inference in resnet-18 model.
+In this example, we use NVIDIA DALI for pre-processing image input for inference in resnet-18 and mnist models.
 
 Refer to [NVIDIA-DALI-Documentation](https://docs.nvidia.com/deeplearning/dali/user-guide/docs/index.html) for detailed information
 
+## DALI Pre-Processing with Default dali_image_classifier handler for resnet-18 model
+
+### Create model-archive file
+
+Navigate to serve folder
+
+:warning: Remove the `pipeline_file` key from the `model-config.yaml` file to use the default dali_image_classifier handler
```

Comment:
It's not clear what this is meant for? 
When do we specific this key and when we shouldnt?

Reply:
For the default built-in pipeline no need to specify the pipeline_file. Will be better to have this as a commented setting in the default yaml and ask user's to uncomment the setting and provide name of pipeline_file if they want to override and replace the default processing with their own logic.